plugins/lua/eventful.lua

```diff
@@ -164,6 +164,7 @@ eventType=invertTable{
     "UNIT_ATTACK",
     "UNLOAD",
     "INTERACTION",
+    "PRESAVE",
     "EVENT_MAX"
```

Comment:
Is there a reason this still uses the old EVENT_MAX name, instead of EVENT_MAX_EM?  For that matter, is there a reason for that rename?